simple_salesforce/tests/test_login.py

```diff
@@ -75,6 +75,6 @@ def test_failure(self):
                 username='myemail@example.com.sandbox',
```

Comment:
Can you add tests here for to ensure:

1. The correct domain is used when specified as a parameter?
2. Both parameters being used will result in some kind of an error?